chore: review the plugin readme (#243)
hamlet-io/engine-plugin-aws: README.md

## Hamlet Deploy Plugin - AWS Provider

This is a Hamlet Deploy plugin repository. It extends the Hamlet Deploy application with integration with the AWS cloud provider.

> This repository also includes a second Hamlet Deploy Plugin, specific to testing the AWS Plugin.

Reviewer: ```suggestion
This repository includes a collection of hamlet plugins specific to AWS
```

Author: I've made this change manually
